eth/withdrawals.go

```diff
@@ -0,0 +1,23 @@
+package eth
+
+import "github.com/INFURA/go-ethlibs/rlp"
+
+type Withdrawal struct {
+	Index          Quantity `json:"index"`
+	ValidatorIndex Quantity `json:"validatorIndex"`
+	Address        Address  `json:"address"`
```

Comment:
The Ethereum JSON-RPC spec names the `address` field `recipient`. https://github.com/ethereum/execution-apis/blob/main/src/schemas/withdrawal.yaml
```suggestion
	Recipient        Address  `json:"recipient"`
```

Reply:
Unclear yet which is the correct term, but looks like it's actually going to be `address` after all: https://github.com/ethereum/execution-apis/issues/378